docs/book/releases.md

```diff
@@ -0,0 +1,57 @@
+<!-- markdownlint-disable MD033 -->
+<!-- markdownlint-disable MD004 -->
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver Releases
+
+- [v2.1.0](#v2.1.0)
+- [v2.0.1](#v2.0.1)
+- [v2.0.0](#v2.0.0)
+- [v1.0.2](#v1.0.2)
+- [v1.0.1](#v1.0.1)
+
+## v2.1.0<a id="v2.1.0"></a>
+
+* Refer https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.1.0 for features listing and notable changes.
+* Known issues with v2.1.0 release
+  * `v2.1.0` has issue with static file volume provisioning. When the file PV is re-created with the same PV name, volume is not getting registered as a container volume with vSphere and thus further operations like attach/delete can not be performed on the volume.
```

Comment:
Have we fixed this? This sounds like a critical bug that needs to be fixed into a patch release.

Reply:
No this is not yet fixed. @chethanv28 is working on fixing this issue.